lib/src/index.js

```diff
@@ -19,6 +19,10 @@ function src(glob, opt) {
     throw new Error('Invalid glob argument: ' + glob);
   }
 
+  // Leverage node-glob's statCache, conservatively
+  // TODO allow users to provide this option, at their own risk?
+  opt.statCache = Object.create(null);
```

Comment:
Ah, right. I figured it might be all right here as it's not a documented option, but yes it would linger on the options object of course. I propose to leave this as is for just now, pending the options refactor you've started.